src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -103,12 +103,25 @@ fi
 	echo $APP_PACKAGES_PATH > $SITE_PACKAGES_PATH"/oryx.pth"
 {{ end }}
 
+packageWheelType={{ PythonPackageWheelProperty }}
+
 {{ if RunPythonPackageCommand }}
 	echo
 	echo "Running python packaging commands ...."
 	echo
-	echo "Creating python package wheel ...."
-	$python setup.py sdist --formats=gztar,zip,tar bdist_wheel --universal
+	echo "Determining python package wheel ...."
+
+	{{ if PythonPackageWheelProperty }}
+		echo "Creating universal package wheel ...."
```

Comment:
Move just before line 123?

Reply:
actually if I keep it like this it's easier to write a test and assert on the content.